tests/python38/test_ast_refactor.py

```diff
@@ -0,0 +1,16 @@
+from sys import version_info
+
+import pytest
+
+import taichi as ti
+
+
+@ti.test(experimental_ast_refactor=True)
```

Comment:
can we add a note here to explain why this is a separate file/link to the github issue? :P

Reply:
I will add it in the next pr.